terraform/gcp/modules/external_secrets/external_secrets.tf

```diff
@@ -42,10 +47,17 @@
   depends_on = [google_service_account.external_secrets_sa]
 }
 
+resource "google_project_iam_member" "external_secrets_binding_token" {
+  project    = var.project_id
+  role       = "roles/iam.serviceAccountTokenCreator"
```

Comment:
Post submit feedback @k4leung4 : can this be updated to grant the role on the specific service accounts(s) that need impersonating? This is a very powerful grant since the external-secrets SA can use access of any other service accounts in the project.

Reply:
@var-sdk good point. created https://github.com/sigstore/scaffolding/pull/264